api/src/main/resources/rdr.yaml

```diff
@@ -281,10 +279,10 @@ definitions:
       - UNDERGRADUATE
       - MASTER
       - DOCTORATE
-      -
+
   Disability:
     type: string
     enum:
-      - YES
-      - NO
+      - "YES"
```

Comment:
There is a bug in Swagger that converts ENUM YES and NO to true and false unless its Explicitly used as string "YES"

Reply:
😠 

Can you please double-check that when this goes through our script to create `merged.yaml` that the quotes are still present? I'm worried they won't be.

In fact, I'm confused Swagger can distinguish the difference; by the time the YAML is loaded all bets are off w/r/t quotes.

Reply:
Since this is in RDR.yaml, merged.yaml doesnt have these changes